repository/src/main/resources/db/changelog/db.changelog-rdm-9338.xml

```diff
@@ -5,7 +5,7 @@
     <changeSet id="rdm-9338" author="kiran.yenigala@hmcts.net">
 
         <createTable tableName="noc_config">
-            <column name="id" type="serial">
+            <column autoIncrement="true" name="id" type="serial">
```

Comment:
we don't need this table, as it is removed.. Please. try to merge the changes form **develop** branch

Reply:
All change log files are deleted any way..by flyway changes,  I have merged the branch with changes from develop?